app.py

```diff
@@ -10,13 +11,55 @@
 @app.route('/get_weather', methods=['POST'])

```

Comment:
Can we change these routes? underscores don't seem very common in URLs, usually its dashes instead but how about instead:
/get_weather -> /forecast/single-day
/get_7_day_forecast -> /forecast/weekly
/get_hourly_forecast -> /forecast/hourly
/get_current_forecast -> /forecast/current or /forecast